packages/balancer/services/autonomous_fund_mainnet/service.yaml

```diff
@@ -0,0 +1,35 @@
+name: autonomous_fund_mainnet
```

Comment:
Sorry to be pedantic, but suggest swapping the test one to `_goerli` and having `autonomous_fund` reserved for the Ethereum mainnet deployment

Reply:
You haven't removed the `_mainnet` yet :)

Reply:
ah missed that part sorry, yeah sure I will remove it now